packages/headless/src/features/fields/fields-actions.ts

```diff
@@ -29,7 +29,7 @@ export const fetchFieldsDescription = createAsyncThunk<
   FieldDescription[],
   void,
   AsyncThunkSearchOptions<ConfigurationSection>
->('fields/description', async (_, {extra, getState, rejectWithValue}) => {
```

Comment:
I think this is breaking if somebody uses a custom reducer. However,  don't know what the chances are of that, and I do prefer the new names 🤔

Reply:
I started by using these but it got confusing. I didn't revert cause in the end I like the new names either

Reply:
It's fine to change this.